plugins/outputs/cloudwatch/README.md

```diff
@@ -45,4 +45,7 @@ also save AWS API cost. If enable this flag, this plugin would parse the require
 [CloudWatch statistic fields](https://docs.aws.amazon.com/sdk-for-go/api/service/cloudwatch/#StatisticSet) 
 (count, min, max, and sum) and send them to CloudWatch. You could use `basicstats` 
 aggregator to calculate those fields. If not all statistic fields are available, 
-all fields would still be sent as raw metrics.
\ No newline at end of file
+all fields would still be sent as raw metrics.
+
+## high_resolution_metrics
```

Comment:
I think this is missing one dash, to make in nest under *Config* section?

Reply:
True, I'm gonna make a new PR to fix it.